src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -235,6 +245,59 @@ public void runAppCfgCommand(List<String> args) throws ProcessRunnerException {
     processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  /**
+   * Returns the version of the Cloud Sdk installation. Unlike other methods in this class that call
+   * gcloud, this method always uses a synchronous ProcessRunner and will block until the gcloud
+   * process returns.
+   *
+   * @throws ProcessRunnerException when process runner encounters an error
+   */
+  public String getVersion() throws ProcessRunnerException {
+    validateCloudSdk();
+
+    SynchronousOutputProcessRunner runner = new SynchronousOutputProcessRunner.Builder().build();
+
+    // gcloud info --format="value(basic.version)"
+    List<String> command = Lists.newArrayList(
+        getGCloudPath().toString(), "info");
+
+    command.addAll(GcloudArgs.get("format", "value(basic.version)"));
+
+    // TODO handle error exit codes
+    runner.run(command.toArray(new String[command.size()]));
+
+    return runner.getProcessStdOut();
+  }
+
+  /**
+   * Returns the list of CloudSdkComponents and their settings, reported by the current gcloud
```

Comment:
Cloud SDK Components (see https://cloud.google.com/sdk/docs/managing-components for spelling)